raiden-dapp/src/locales/en.json

```diff
@@ -362,7 +362,10 @@
     }
   },
   "select-hub": {
-    "select-button": "Select Hub"
+    "select-button": "Select Hub",
+    "service-token-tooltip": "Mint and deposit {token} to the User Deposit Contract to pay for the Monitoring Service.",
```

Comment:
do you think this would make more sense under `service-token": { "tooltip": ... }`?

Reply:
I'm not sure, it will only get displayed on the "Select Hub" view and nowhere else

Reply:
I would only probably use it just for grouping